editor/editor_help.cpp

```diff
@@ -1782,7 +1782,7 @@ void EditorHelpBit::_bind_methods() {
 void EditorHelpBit::_notification(int p_what) {
 
 	if (p_what == EditorSettings::NOTIFICATION_EDITOR_SETTINGS_CHANGED) {
-		add_style_override("panel", EditorNode::get_singleton()->get_gui_base()->get_stylebox("ScriptPanel", "EditorStyles"));
+		add_style_override("panel", get_stylebox("ScriptPanel", "EditorStyles"));
```

Comment:
[Line 1802](https://github.com/toger5/godot/blob/00791b647c9b97f29ebe63c02a6fc7d4deb8037a/editor/editor_help.cpp#L1802) should have this fix as well. :)